web/src/desktop_notifications.ts

```diff
@@ -0,0 +1,107 @@
+import $ from "jquery";
+
+import type {ElectronBridge} from "./electron_bridge";
+import type {Message} from "./message_store";
+
+const electron_bridge: ElectronBridge | undefined = window.electron_bridge;
+class ElectronBridgeNotification extends EventTarget {
+    constructor(title: string, options: NotificationOptions) {
+        super();
+        if (
+            electron_bridge &&
+            electron_bridge.new_notification &&
+            typeof electron_bridge.new_notification === "function"
```

Comment:
`&& typeof electron_bridge.new_notification === "function"` is an unnecessary addition. If the value is defined, it will be a function, and TypeScript knows that already from the type of `electron_bridge`.

Reply:
Thank You for pointing it out.
I will remove that.